.pyproject_generation/pyproject_custom.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "hexkit"
-version = "2.1.0"
+version = "2.2.0"
```

Comment:
I think it is rather a bugfix, right? We though it would work like that in the first place. Now we match the expectations.